migrations/1_initialize_schema.up.sql

```diff
@@ -241,66 +241,6 @@ CREATE UNIQUE INDEX unique_global_integration_name ON integration (name, type)
 CREATE UNIQUE INDEX unique_project_integration_name ON integration (name, type, project_id)
     WHERE project_id IS NOT NULL;
 
--------------------------------- LDAP configurations ------------------------------
```

Comment:
are going to put it into v5.1
it means that we will have already running database
so, it is needed to be done in separate migration script with config data migration